src/js/app.js

```diff
@@ -55,4 +51,14 @@ document.addEventListener("turbolinks:load", function () {
   document.querySelectorAll('pre code').forEach((block) => {
     hljs.highlightElement(block);
   });
+
+  let autosaveKeyupTimer
+  document.addEventListener('keyup', (event) => {
+    if (event.target.form.classList.contains('autosave')) {
+      clearTimeout(autosaveKeyupTimer)
+      autosaveKeyupTimer = setTimeout(() => {
+        event.target.form.requestSubmit()
+      }, 1000)
+    }
+  })
```

Comment:
What is this needed for?

Reply:
https://github.com/luckyframework/website/pull/949#issuecomment-1061488005
I can drop this diff, but it was an easy add with turbo involved

Reply:
It's probably ok. I'm just trying to understand what is required for turbo to work out of the box. We're using this as a testing ground before adding it in to Lucky core. So I don't want to confuse what's absolutely needed vs what's sugar.

Reply:
Ignore the secondary commit for the fluff
The only thing required for the turbo form to post to itself is the frame so it knows what to update